src/main/java/seedu/type/SpendingList.java

```diff
@@ -24,18 +25,15 @@ public SpendingList() {
         dailyBudget = 0;
         monthlyBudget = 0;
         yearlyBudget = 0;
+        total = 0;
```

Comment:
actually can initialise from the start?

Reply:
As in together with the declaration?  I do this to follow the budget but yes it can be done outside.